src/graph/GraphModel.ts

```diff
@@ -111,16 +112,20 @@ export class GraphModel {
     )
   }
 
-  getNode(id: string): DecoratedNode {
-    return this.decoratorModel.getDecoratedNodes([
-      this.contentModel.getNode(id),
-    ])[0]
+  getNode(id: string): DecoratedNode | undefined {
+    const node = this.contentModel.getNode(id)
```

Comment:
Some things here use `contentModel.getNode(id)` others `contentModel.nodes[id]`
ideally it would all be one or the other